pkg/security/consts/consts.go

```diff
@@ -20,6 +20,9 @@ const (
 	// AppAPITokenEnvVar is the environment variable for the app API token.
 	//nolint:gosec
 	AppAPITokenEnvVar = "APP_API_TOKEN"
```

Comment:
If we change this to be prefixed with DAPR_, we will need to support both and follow the deprecation path. This env is passed by the user, optionally. So, it is part of the contract.